### DISCUSSION TITLE
[请教] web端 Class Matrix里面的setRotate方法。怎么调用它？

### DISCUSSION THREAD

**Discussion by leohandsome1688** [Tencent/libpag#2017] [Q&A] [answered]
【请教】 web端 Matrix类里面的setRotate方法。怎么调用它？尝试了很多方式，但是都不行，

**Comment by zhalice2011** [comment, marked as answer]
你好, 手动设置 Matrix demo 如下: 

预览: https://pagtest.zhalice.com/pages/scale-mode-background-after.html

源代码: view-source:https://pagtest.zhalice.com/pages/scale-mode-background-after.html


setRotate 没有示例, 我试了一下是可以正常使用的

```js
scaleMatrix = PAG.Matrix.makeScale(1 * devicePixelRatio, 1 * devicePixelRatio);
  
scaleMatrix.setRotate(30)

pagView.setMatrix(scaleMatrix);

await pagView.play();
```
